lunatrace/bsl/backend/src/models/vulnerability-dependency-tree/graph.ts

```diff
@@ -12,33 +12,33 @@
  *
  */
 export class Graph {
-  private forwardEdges: Map<string, string> = new Map();
-  private backwardEdges: Map<string, Set<string>> = new Map();
+  private childEdges: Map<string, string> = new Map();
+  private parentEdges: Map<string, Set<string>> = new Map();
```

Comment:
this should probably be 'childIdToParentIds'

Reply:
yeah, i could see that luckily. Luckily when you go to use it reads like `parentEdges(childId)` which is pretty explanatory